src/main/java/uk/gov/hmcts/reform/sscs/client/CommonReferenceDataApi.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.hmcts.reform.sscs.client;
+
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+import static org.springframework.http.HttpHeaders.CONTENT_TYPE;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+import java.util.Optional;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import uk.gov.hmcts.reform.sscs.model.CaseFlag;
+import uk.gov.hmcts.reform.sscs.model.Categories;
+import uk.gov.hmcts.reform.sscs.model.CategoryRequest;
+
+@FeignClient(
+    name = "common-ref-data-api",
+    url = "${common-ref.api.url}"
+)
+public interface CommonReferenceDataApi {
+    String SERVICE_AUTHORIZATION = "serviceAuthorization";
+
+    @RequestMapping(
+        method = RequestMethod.GET,
+        value = {"refdata/commondata/lov/categories", "refdata/commondata/lov/categories/{categoryId}"},
+        headers = CONTENT_TYPE + "=" + APPLICATION_JSON_VALUE
+    )
+    ResponseEntity<Categories> retrieveListOfValuesByCategoryId(
+        @RequestHeader(AUTHORIZATION) String authorisation,
+        @RequestHeader(SERVICE_AUTHORIZATION) String serviceAuthorization,
+        @PathVariable(value = "categoryId") Optional<String> categoryId,
+        CategoryRequest categoryRequest
```

Comment:
Should this have @RequestBody annotation?

Reply:
This is the endpoint we are trying to hit, in the method signature there is no requestBody https://github.com/hmcts/rd-commondata-api/blob/4e675ebeca8b1adb193064478f0631eaf1877719/src/main/java/uk/gov/hmcts/reform/cdapi/controllers/CrdApiController.java